tests/src/service_road_loop/geometry.json

```diff
@@ -5,86 +5,36 @@
         "coordinates": [
```

Comment:
The diff here is due to updating `geom`.
before:
![Screenshot from 2022-09-04 20-00-24](https://user-images.githubusercontent.com/1664407/188329725-dea6b77a-0a56-4aef-9715-f1591937445f.png)
That road polygon is expressed as a multipolygon with two triangles, which is quite silly.
after:
![Screenshot from 2022-09-04 20-00-49](https://user-images.githubusercontent.com/1664407/188329740-4eed3a0c-3ffc-40e7-b43b-805ddbe0c1c7.png)
The fix comes from https://github.com/a-b-street/abstreet/issues/951, which is an epic saga to make `geom::Polygon` always represent a "good" polygon composed of an exterior ring and optional interior rings.